src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule731selfexplanatory/InputJavadocMethodAndMissingJavadocMethod.java

```diff
@@ -138,6 +138,32 @@ String foo91() {
   public String foo92() {
     return "Fooooo" + "ooo" + "ooooooo" + "ooooo" + "ooo";
   }
+
+  int xyz = 99;
+
+  public int getXyz() {
+    return xyz;
+  }
+
+  public void setXyz(int xyz) {
+    this.xyz = xyz;
+  }
+
+  int abc = 101;
+
+  // violation below 'Missing a Javadoc comment.'
+  public int getAbc() { // this will not work
+    abc = 55;
+    abc += 1;
+    return abc;
+  }
+
+  // violation below 'Missing a Javadoc comment.'
+  public void setAbc(int abc) { // this will not work
+    abc = 55;
+    abc += 1;
+    this.abc = abc;
+  }
```

Comment:
these types of getters/setters will not work, from [Check's](https://checkstyle.sourceforge.io/checks/javadoc/missingjavadocmethod.html) documentation:

```
For getters and setters for the property allowMissingPropertyJavadoc, the methods must match exactly the structures below.

public void setNumber(final int number)
{
    mNumber = number;
}

public int getNumber()
{
    return mNumber;
}

public boolean isSomething()
{
    return false;
}
```

if the getter/setter matches only specified type mentioned in documentation only then config allows for missing javadoc otherwise it will give violation.